index1.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+
+
+<body bgcolor=white>
+    
+
+<section>
+    <img src="feedback-banner.jpg" style="width:100%;height:500px;">
+    <form action="/action_page.php">
+ Username:<br>
+  <input type="text" name="username" value="">
+  <br>
+ Password:<br>
+  <input type="text" name="password" value="">
+  <br><br>
```

Comment:
Similar to my comment on L5, tags such as `<br>`, `<b>`, `<i>`, `<u>` have all but lost their presentational value in HTML5, with their semantic meaning considerably watered down from what they used to represent in previous versions of HTML. It is best to rely on CSS to inform the browser how a document should look and only keep information pertaining to what data to display in the HTML.